packages/starters/backend/sirius-components-task-starter/src/main/java/org/eclipse/sirius/components/task/starter/configuration/AbstractTaskPropertiesConfigurer.java

```diff
@@ -139,58 +151,96 @@ private List<AbstractControlDescription> getGeneralControlDescription() {
         return controls;
     }
 
-    private TextfieldDescription getEndTimeWidget() {
-        var endTime = this.propertiesWidgetCreationService.createTextField("abstractTask.endTime", "End Time",
-                task -> {
-                    Instant endInstant = ((AbstractTask) task).getEndTime();
-                    try {
-                        return DateTimeFormatter.ISO_INSTANT.format(endInstant);
-                    } catch (DateTimeException  | NullPointerException e) {
-                        // Ignore
-                    }
-                    return "";
-                },
-                (task, newValue) -> {
-                    if (newValue != null && newValue.isBlank()) {
-                        ((AbstractTask) task).setEndTime(null);
-                    } else {
+    private DateTimeDescription getStartTimeWidget() {
+        Function<VariableManager, String> valueProvider = variableManager -> {
+            return variableManager.get(VariableManager.SELF, Object.class)
```

Comment:
Use `variableManager.get(VariableManager.SELF, AbstractTask.class)` instead, this way you are not casting anything with testing it's type first and you don't need to cast at all.